index.js

```diff
@@ -168,19 +168,28 @@ module.exports = function (buf) {
 		};
 	}
 
-	// needs to be before the `webm` check
-	if (buf[31] === 0x6D && buf[32] === 0x61 && buf[33] === 0x74 && buf[34] === 0x72 && buf[35] === 0x6f && buf[36] === 0x73 && buf[37] === 0x6B && buf[38] === 0x61) {
-		return {
-			ext: 'mkv',
-			mime: 'video/x-matroska'
-		};
-	}
-
+	// https://github.com/threatstack/libmagic/blob/master/magic/Magdir/matroska
 	if (buf[0] === 0x1A && buf[1] === 0x45 && buf[2] === 0xDF && buf[3] === 0xA3) {
-		return {
-			ext: 'webm',
-			mime: 'video/webm'
-		};
+		const sliced = buf.subarray(4, 4 + 4096);
```

Comment:
The reason why I used TypedArray methods is because this project can be used from the browser too, so I wanted to not tie it to node.js. Also, subarray creates a view over the underlying data, while slice copies the bytes, which is slower and should be unnecessary.

Regarding the number of bytes the user needs to read, I thought it would be OK to use the same amount as we already do, because it seems to work on the fixtures and all the MKV/WebM files that I tested, and instead we can increase it if someone encounters a file that happens to fail. However if you want, sure I can increase it.